repository/t.json

```diff
@@ -1680,6 +1680,16 @@
 			]
 		},
 		{
+			"name": "Tickscript",
+			"details": "https://github.com/mattyjones/Tickscript",
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"tags": true
```

Comment:
Currently you don't have any [semver](https://semver.org) tags at https://github.com/mattyjones/Tickscript/tags